dev_tools/run_doctest.py

```diff
@@ -192,14 +191,15 @@ def import_file(file_path: str) -> ModuleType:
 
     Returns: The imported module.
     """
-    mod_name = os.path.basename(file_path)
-    if mod_name.endswith('.py'):
-        mod_name = mod_name[:-3]
+    mod_name = 'cirq_doctest_module'
```

Comment:
This doesn't look right: we're passing "cirq_doctest_module" as module name regardless of what module we're actually importing.

Reply:
This is what it was doing before.  It used to import "filename" as a top-level module.  It imports each file as if it is an independent script so it doesn't interfere with the previously imported `cirq` module.